.github/workflows/build-package-docs.yaml

```diff
@@ -26,19 +26,13 @@ on:
         - japanese
       ansible-package-version:
         type: choice
-        description: Version of the Ansible community package to build
+        description: Ansible community package version
         required: true
         default: devel
         options:
         - devel
         - '10'
```

Comment:
Consider adding '11' here. We did that on the jenkins side typically as soon as the prior version (aka Ansible 10) was released.